src/views/penaltydetails.njk

```diff
@@ -0,0 +1,100 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk"  import govukBackLink %}
+{% from "govuk/components/input/macro.njk"      import govukInput %}
+{% from "govuk/components/details/macro.njk"    import govukDetails %}
+{% from "govuk/components/button/macro.njk"     import govukButton %}
+
+
+{% block pageTitle %}
+  Tell us the penalty details
+{% endblock %}
+
+{% block beforeContent %}
+  {{ 
+    govukPhaseBanner({
+      tag: {
+        text: "alpha"
+      },
+      html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+    }) 
+  }}
+  {{ 
+    govukBackLink({
```

Comment:
Shall we quickly remove it as it is about of scope of the story? Same as phase banner?

Reply:
agreed, although the layout.njk includes a Beta banner. Do you suggest we remove it from there aswell?